src/Tokenizer.php

```diff
@@ -770,24 +770,24 @@ public function __construct()
         };
 
         // Set up regular expressions
-        $this->regexBoundaries       = '(' . implode(
+        $this->regexBoundaries       = '(?>' . implode(
```

Comment:
I don't know this notation. Where can I find docs on this?

Reply:
We do not need/want to backtrack in the regular expressions, so we use "atomic group" - https://www.regular-expressions.info/atomic.html